### PR TITLE
Add proxies for row and RowList

### PR DESCRIPTION
**Pull request by LiamGoodacre**

### CODE REVIEW

**Comment by LiamGoodacre** [comment]
For #13.

File: src/Type/Row.purs [hunk 1]
```diff
@@ -43,6 +47,8 @@ foreign import kind RowList
 foreign import data Nil :: RowList
 foreign import data Cons :: Symbol -> Type -> RowList -> RowList
 
+data RLProxy (rowList :: RowList) = RLProxy
```
Comment (paf31) [inline src/Type/Row.purs]: Do you think it'll be useful to have value-level equivalents for `RowToList` and `ListToRow`?

**Comment by paf31** [comment]
👍 LGTM, thanks!

File: src/Type/Row.purs (hunk 1 above)
Comment (LiamGoodacre) [inline src/Type/Row.purs]: To repeat what I said on slack: I'm tempted to leave them out. I think they're only use was cutting down occurrences of `Proxy :: Proxy t`, but the new proxy syntax is better for that.  I've had a look through uses of typelevel-prelude on github and I couldn't see any usages of any of the value-level equivalents.

**Pull request merged**